driver/config/config.go

```diff
@@ -66,6 +66,7 @@ const (
 	ViperKeyCourierSMTPClientKeyPath                         = "courier.smtp.client_key_path"
 	ViperKeyCourierTemplatesPath                             = "courier.template_override_path"
 	ViperKeyCourierTemplatesRecoveryInvalidEmail             = "courier.templates.recovery.invalid.email"
+	ViperKeyCourierTemplatesRecoveryInvalidSend              = "courier.templates.recovery.invalid.send"
```

Comment:
I think this key should go in another object because here we are configuring the templates. Given that this affects recovery and the `link` and `code` strategy, it should probably be configured there?

For completeness it would also be good to be able to disable this for both recovery and verification.